test/apps/react-oie/vite.config.js

```diff
@@ -26,6 +26,9 @@ export default defineConfig({
     port: 8080
   },
   build: {
+    // Minimize transpile in development
+    target: process.env.NODE_ENV === 'development' ? 'esnext' : 'modules',
```

Comment:
all build related configuration only apply to `vite build`, which I added to the `start` script in my PR. 
https://github.com/okta/okta-auth-js/pull/1144/files#diff-49de8fcd34f4ac365d845b18c0fcb755aa3e266cd94693e799629b376631b047